plugin/sexy_scroller.vim

```diff
@@ -62,9 +66,11 @@ command! SexyScrollerToggle call s:ToggleEnabled()
 
 augroup Smooth_Scroller
   autocmd!
-  autocmd CursorMoved * call s:CheckForChange(1)
-  autocmd CursorMovedI * call s:CheckForChange(1)
-  autocmd InsertLeave * call s:CheckForChange(0)
+  " Wrap all commands (as strings) with the cmd wrapper so they can be
+  " turned on or off with the g:SexyScroller_AutcmdsEnabled option
+  autocmd CursorMoved * call s:AutocmdCmdWrapper("call s:CheckForChange(1)", g:SexyScroller_AutocmdsEnabled)
+  autocmd CursorMovedI * call s:AutocmdCmdWrapper("call s:CheckForChange(1)", g:SexyScroller_AutocmdsEnabled)
+  autocmd InsertLeave * call s:AutocmdCmdWrapper("call s:CheckForChange(0)", g:SexyScroller_AutocmdsEnabled)
```

Comment:
I would be tempted not to pass the second argument `a:run` here, and let the function pick up `g;SexyScroller_AutocmdsEnabled`.

But perhaps you envisage calling this function with a different argument in future...?  :)